adr/ADR-20.md

```diff
@@ -98,6 +100,7 @@ type ObjectStoreConfig struct {
 	Storage     StorageType		// stream storate_type
 	Replicas    int			// stream replicas
 	Placement   Placement		// stream placement
+	Compression  bool		// stream compression, s2 or none
```

Comment:
its not `s2 or none`

Reply:
The missing verbiage I mention is something like ``` * If compression is requested in the configuration set compression to s2``` that was added to kv adr

Reply:
Yeah when implementing KV devs were confused thats why it was added, so just pre-empting.